modules/metastore/metastore.install

```diff
@@ -114,3 +114,26 @@ function metastore_update_8007() {
   $config = \Drupal::service('config.factory')->getEditable('node.type.data');
   $config->set('preview_mode', 1)->save();
 }
+
+/**
+ * Update existing data dictionary nodes to use corrected schema.
+ */
+function metastore_update_8008() {
+  $ids = \Drupal::service('dkan.metastore.service')->getIdentifiers('data-dictionary');
+  $count = 0;
+  foreach ($ids as $id) {
+    $dict = \Drupal::service('dkan.metastore.metastore_item_factory')->getInstance($id);
+    $metadata = $dict->getMetadata();
+    if (!isset($metadata->data->title) && isset($metadata->title)) {
+      $metadata->data->title = $metadata->title;
+      unset($metadata->title);
+    }
+    $dict->setMetadata($metadata);
+    $dict->save();
+    $count++;
+  }
+  return t("Updated $count dictionaries. If you have overridden DKAN's core schemas, 
+    you must update your site's data dictionary schema after this update. Copy
+    modules/contrib/dkan/schema/collections/data-dictionary.json over you local
```

Comment:
From `modules/contrib/dkan/schema/collections/data-dictionary.json over you local` to `modules/contrib/dkan/schema/collections/data-dictionary.json over to your local`

Reply:
I'm wondering if we should add a revision row with a note of what changed?